descriptors/annotators/detection/PlaneAnnotator.xml

```diff
@@ -72,6 +80,13 @@
         </value>
       </nameValuePair>
 
+      <nameValuePair>
+        <name>use_non_nan_cloud</name>
+        <value>
+          <string>false</string>
```

Comment:
found the probelm; this should be <boolean> tag not <string>

Reply:
yeah this is a hilarious mistake :). Should be tag boolean

Reply:
I fixed